### ISSUE TITLE
[Request] always block domains for exotic schemes per default

### ISSUE THREAD

**Issue by N4melessGitter** [gorhill/uMatrix#523]
the Add-ons Manager in Firefox (about:addons) falls under the whitelisted about-scheme and connects to https://ssl.google-analytics.com/ga.js.
I know I could un-whitelist the whole about-scheme but that would require a lot of specific rules
for every sub-category, f.e.:
`addons.about-scheme * * allow`
It would be easier and hopefully it's possible to always block the domains from the blocklists in those whitelisted schemes per default.
I don't think that only blocking those "bad" domains would break anything, unless someone adds mozilla.org or something to their blocklists.
Basically like "noop" in uBlock for those scheme's.
Any better ideas how to handle this correctly are welcome. Thank you

**Comment by gorhill** [comment]
I don't understand the request -- `about-scheme * * allow` seems to accomplish what you want.

**Comment by gorhill** [comment]
uMatrix is entirely based on rules, I do not intend to create exceptions to this -- I want uMatrix's behavior to be predictable.
